src/main/java/org/klodnicki/entity/Account.java

```diff
@@ -0,0 +1,39 @@
+package org.klodnicki.entity;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "accounts")
+public class Account {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    private Long id;
+    @Column(nullable = false)
```

Comment:
Add property `name` to the `@Column` to follow database convention of naming columns.
i.e. `@Column(name = "first_name")`

Reply:
@BartoliniAndBorderCollies great job on this issue! Done.